<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Option</class>
 <widget class="QDialog" name="Option">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="CloseBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="CPerformaceTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;
border:1px solid #ccc;</string>
   </property>
   <property name="text">
    <string>Perfomance</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ColorPreviewSize">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>1080</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="SaveAsk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Don't ask when saving.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ResizeRate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Do not keep rates automatically.</string>
   </property>
  </widget>
  <widget class="QLabel" name="ColorPreviewSizeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB Color Preview Size [Heignt](CPU)</string>
   </property>
  </widget>
  <widget class="QLabel" name="SaveTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;
border:1px solid #ccc;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="ResizeTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;
border:1px solid #ccc;</string>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
  </widget>
  <widget class="QLabel" name="ThemeTab">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;
border:1px solid #ccc;</string>
   </property>
   <property name="text">
    <string>Theme</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ThemeBlack">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#222;
border-width: 1px;
border-style: solid;
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ThemeWhie">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fff;
border-width: 1px;
border-style: solid;
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ThemePurple">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#993ECC;
border-width: 1px;
border-style: solid;
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>Purple</string>
   </property>
  </widget>
  <widget class="QLabel" name="CUndoSizeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of 'undo' [Restart required](Memory)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="CUndoSize">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SaveCompress">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="ColorPreviewSizeLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Compressibility(only .jpg)</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>SaveAsk</tabstop>
  <tabstop>CloseBtn</tabstop>
  <tabstop>ResizeRate</tabstop>
  <tabstop>ThemePurple</tabstop>
  <tabstop>ThemeWhie</tabstop>
  <tabstop>ThemeBlack</tabstop>
  <tabstop>ColorPreviewSize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
